deployment/deploy/DOMAIN_NAME.sdk

```diff
@@ -0,0 +1 @@
+testing.exaworks.org
```

Comment:
just wondering is this need to be reflected in config files? (`deployment/docker/fs/etc/psij-testing-service` -> `config-*.json`)

Reply:
In principle, yes. There is supposed to be a manual "check and edit configuration files" step that should take care of it. We could automate that, but it's not strictly necessary.

Reply:
I see, ok, thanks!